Dockerfile

```diff
@@ -1,23 +1,16 @@
-FROM centos:7
+FROM manageiq/ruby
 ENV container docker
-MAINTAINER ManageIQ https://github.com/ManageIQ/manageiq-appliance-build
```

Comment:
I think we should override the maintainer for this image (I would make it the URL of this repo)